controllers/score-controllers.js

```diff
@@ -0,0 +1,58 @@
+const { Hospital, Score } = require('../db/models');
+const Sequelize = require('sequelize');
+
+exports.sendHospitalScore = function(req, res) {
+  const hospitalId = req.query.hospitalId;
+  const score = req.query.score;
+  const userid = req.query.userId;
+  Score.create({
+    rate: score,
+    UserId: userid,
+    HospitalId: hospitalId,
+  }).then(() => {
+    try {
+      const Op = Sequelize.Op;
+      Score.findAll({
```

Comment:
Since this is a promise, we don't need to put everything under `try`, we can catch the promise and handle the error there:
```js
Score.findAll(...)
  .catch(e => {
// Here we put the code we've used in the `catch` block
  })
```